File: RealEstateManager/app/src/main/java/com/openclassrooms/realestatemanager/utils/TestList.java
package com.openclassrooms.realestatemanager.utils;

import com.openclassrooms.realestatemanager.models.Address;
import com.openclassrooms.realestatemanager.models.Estate;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collections;
import java.util.List;

public abstract class TestList {

    private static final List<Integer> photos = Collections.emptyList();
    private static final Address address = new Address(100,"Fake Street","Additional Address","Fake Country","000000");

    private static final List<Estate> estateList = Arrays.asList(
            new Estate(photos, "Fake_category", "000 000 €", "Fake_Description", 600, 8, 1, 2, address)
    );

    public static ArrayList<Estate> getTestList() {
        return new ArrayList<>(estateList);
    }
}

File: RealEstateManager/app/src/main/java/com/openclassrooms/realestatemanager/testList.kt
package com.openclassrooms.realestatemanager

import com.openclassrooms.realestatemanager.models.Address
import com.openclassrooms.realestatemanager.models.Estate
import kotlin.collections.ArrayList


object TestList {
    private val photos = mutableListOf<Int>(
        R.drawable.apartment,
        R.drawable.living_room,
        R.drawable.kitchen,
        R.drawable.bathroom,
        R.drawable.bedroom
    )

    private val address =
        Address(100, "Fake Street", "Additional Address", "Fake Country", "000000")

    private val estateList = mutableListOf<Estate>(
        Estate(photos, "Fake_category 1", "700 000 €", "Fake Description", 600, 8, 1, 2, address),
        Estate(photos, "Fake Category 2", "800 000 €", "Fake_Description", 800, 9, 2, 3, address)
    )
    val getTestList: ArrayList<Estate>
        get() = ArrayList(estateList)
}



